data/scripts/systems/outfits/data/mounts.lua

```diff
@@ -0,0 +1,272 @@
+-- Mount data source for the Outfits & Mounts module.
+--
+-- This file overrides the Game mount lookup helpers used throughout the module:
+-- - Game.getMounts()
+-- - Game.getMountByLookType(lookType)
+-- - Game.getMountByName(name)
+-- - Game.getMount(param)
+--
+-- Each entry is keyed by lookType and includes:
+-- - name (string)
+-- - speed (number)
+-- - premium (bool)
+local mounts = {
+    [368] = { name = "Widow Queen", speed = 20, premium = true },
```

Comment:
I suggest  to keep sequential mount IDs and use that instead of the looktype as an ID